docs/ops/movement/ScatterElementsUpdate_12.md

```diff
@@ -0,0 +1,209 @@
+# ScatterElementsUpdate {#openvino_docs_ops_movement_ScatterElementsUpdate_12}
+
+@sphinxdirective
+
+**Versioned name**: *ScatterElementsUpdate-12*
+
+**Category**: *Data movement*
+
+**Short description**: Creates a copy of the first input tensor with elements updated according to the logic specified by *reduction* attribute and the following inputs: ``indices``, ``updates``, ``axis``.
+
+**Detailed description**: For each element of ``updates`` input, the value of ``indices`` at the same index points the element to update on the ``axis`` of ``data``. The index for dimension pointed by ``axis`` is provided by values of ``indices`` input, otherwise the index is the same as the index of the entry itself.
```

Comment:
```suggestion
**Detailed description**: For each element of ``updates`` input, the value of ``indices`` at the same index points the element to update on the ``axis`` of ``data``. The index for dimension pointed by ``axis`` is provided by values of ``indices`` input, otherwise, the index is the same as the index of the entry itself.
```